### PR TITLE
DR-1111 Change how slider renders steps and change plan by name

### PR DESCRIPTION
**Pull request by mvillaseco**
# Overview
Small Fixes

# Changes:
 - Change Slider selector to use Name plan instead of hardcoded id.
 - Remove links to My-Plan Page.
 - Add drag me icon in spanish.

### CODE REVIEW

File: src/wwwroot/controllers/PlanCtrl.js [hunk 1]
```diff
@@ -22,50 +22,45 @@
...
     var planItems;
+    var planItemsParsedForSlider;
+    vm.langUsed = $translate.use();
 
     function activate() {
       return settings.getPlansAvailable().then(function(response){
-        planItems = response.data;
-        changePlan(defaultPlanId);
+        planItems = response.data.plans;
+        planItemsParsedForSlider = response.data.plans.map(function(plan){
```
Comment (andresmoschini) [inline src/wwwroot/controllers/PlanCtrl.js]: What do you think about removing `planItemsParsedForSlider` field and apply this logic inside `loadSlider()`?

**Comment by andresmoschini** [comment]
Looks really nice, do you have a screenshot or gif to share with us?

**Comment by mvillaseco** [comment]
Screen:

File: src/wwwroot/controllers/PlanCtrl.js [hunk 2]
```diff
@@ -22,50 +22,44 @@
...
-    var defaultPlanId = 6;
+    var defaultPlanName = 'PLAN_60K';
     var planItems;
+    vm.langUsed = $translate.use();
 
     function activate() {
       return settings.getPlansAvailable().then(function(response){
-        planItems = response.data;
-        changePlan(defaultPlanId);
+        planItems = response.data.plans;
```
Comment (nreal) [inline src/wwwroot/controllers/PlanCtrl.js]: I think that `response.data.plans` should be `response.data.items`

Comment (mvillaseco) [inline src/wwwroot/controllers/PlanCtrl.js]: Are you planning on renaming that from the backend?

Comment (mvillaseco) [inline src/wwwroot/controllers/PlanCtrl.js]: Ok, since we are going to change plans to items, i'll change it here also.

**Pull request merged**